src/sage/graphs/generators/families.py

```diff
@@ -539,8 +533,8 @@ def BalancedTree(r, h):
 
     Plot a balanced tree of height 5, whose root node has degree `r = 3`::
 
-        sage: G = graphs.BalancedTree(3, 5)                                             # needs networkx
-        sage: G.show()                          # long time                             # needs networkx sage.plot
+        sage: G = graphs.BalancedTree(3, 5)
+        sage: G.show()                          # long time                             # needs sage.plot
```

Comment:
```suggestion
        sage: G.plot()                          # long time                             # needs sage.plot
```
This would provide a more descriptive output.

Reply:
done